recipe/contao.php

```diff
@@ -0,0 +1,73 @@
+<?php
+namespace Deployer;
+
+require_once __DIR__ . '/symfony.php';
+
+add('recipes', ['contao']);
+
+add('shared_files', ['config/parameters.yml']);
+
+add('shared_dirs', [
+    'assets/images',
+    'contao-manager',
+    'files',
+    'public/share',
+    'system/config',
+    'var/backups',
+    'var/logs',
+]);
+
+set('bin/console', function () {
+    return '{{release_or_current_path}}/vendor/bin/contao-console';
+});
+
+set('contao_version', function () {
+    return run('{{bin/console}} contao:version');
+});
+
+// The public path is the path to be set as DocumentRoot and is defined in the `composer.json` of the project
+// but defaults to `public` from Contao 5.0 on.
+// This path is relative from the {{current_path}}, see [`recipe/provision/website.php`](/docs/recipe/provision/website.php#public_path).
+set('public_path', function () {
+    $composerConfig = json_decode(file_get_contents('./composer.json'), true, 512, JSON_THROW_ON_ERROR);
+
+    return $composerConfig['extra']['public-dir'] ?? 'public';
+});
+
+// This task updates the database. A database backup is saved automatically as a default.
+desc('Run Contao migrations');
+task('contao:migrate', function () {
+    run('{{bin/php}} {{bin/console}} contao:migrate {{console_options}}');
+});
+
+// Optional task to download the `contao-manager.phar.php` in the public path. Make sure to
+// set a password on first access!
+desc('Download the Contao Manager');
+task('contao:manager:download', function () {
+    run('curl -LsO https://download.contao.org/contao-manager/stable/contao-manager.phar && mv contao-manager.phar {{release_or_current_path}}/{{public_path}}/contao-manager.phar.php');
```

Comment:
I have no clue, what this file is, but if it lives in public, maybe delete it after deployment?

Reply:
This file is definitely there to stay :-) Can be reached under https://example.org/contao-manager.phar.php.